nevergrad/functions/photonics/test_core.py

```diff
@@ -171,8 +171,29 @@ def test_photosic_reference() -> None:
 
 
 def test_photosic_realist() -> None:
-    eps_and_d = np.array([2.0000, 3.0000, 2.1076, 2.0000, 3.0000, 2.5783, 2.0000, 3.0000,
-                          2.0000, 3.0000, 90.0231, 78.9789, 72.8369, 99.9577, 82.7487,
-                          62.7583, 104.1682, 139.9002, 93.3356, 75.6039])
-    cf_test = photonics.cf_photosic_realist(eps_and_d)
+    eps_and_d = np.array(
+        [
+            2.0000,
+            3.0000,
+            2.1076,
+            2.0000,
+            3.0000,
+            2.5783,
+            2.0000,
+            3.0000,
+            2.0000,
+            3.0000,
+            90.0231,
+            78.9789,
+            72.8369,
+            99.9577,
+            82.7487,
+            62.7583,
+            104.1682,
+            139.9002,
+            93.3356,
+            75.6039,
+        ]
+    )
+    cf_test = photonics.cf_photosic_realistic(eps_and_d)
```

Comment:
@Ellawin @AnMoreau if I get it well, this is structured with the first part being epsilon and the second some kind of thikness? like bragg? Any number of layers work? what are the ones you tend to use?
I'll need to know this so we can explicitely define how this function must be called in the benchmarks

Reply:
It's exactly that for _cf_photosic_realistic_, _eps_and_d_ is a vector composed in a first part with the epsilon values (the material used in each one of the layers), and in a second part with the thicknesses of each one of the layers, like in Bragg. 
Any number of layers can work. Basically I used between 4 and 50 layers, and the best results are generally obtained when the structure has between 10 and 20 layers. The epsilon values are generally comprised between 1.00 and 9.00.

For _cf_photosic_reference_, the vector _X_ is only the thicknesses of each layers, because the materials (so the epislon) are imposed by the function. This is similar in the chirped function.

Reply:
I added these information in the docstring of the functions (the small texts under the "def" line, which is what appears if you look for help on a function)